### DISCUSSION TITLE
Rozwój/utrzymanie filtrów AlleBlock

### DISCUSSION THREAD

**Discussion by MajkiIT** [MajkiIT/polish-ads-filter#17400] [Q&A]
@seba2282 @krystian3w @hawkeye116477 @maciejtarmas 
Powstała dyskusja aby przejąć to a dokładniej kontynuować. Na pewno będzie potrzebne wsparcie aby przepiął linki do filtrów na swojej stronie itp o ile na pewno nie wróci do tematu filtrów.

**Comment by seba2282** [comment]
@MajkiIT dziękuję, że założyłeś taki temat. Teraz ciekawe, kiedy wypowie się @maciejtarmas z oficjalnym stanowiskiem w tej sprawie. 🙂

**Comment by hawkeye116477** [comment]
Ten pan coś już dawno nic nie pisał, rozpłynął się w powietrzu zupełnie jak azet, coś to podejrzane, może Alledrogo mu zalazło za skórę...

Jeżeli o mnie chodzi, to w zasadzie wypowiadałem już się, że ja raczej się na to nie piszę. No chyba, że ty @MajkiIT chcesz, choć widzę, że też czasu ci brakuje na rozwój listy reklamowej, więc raczej to nie najlepszy pomysł...

**Reply by MajkiIT** [reply]
Nie mówię, że ja bo wiesz, jak u mnie, ale może @seba2282 by się zainteresował.

**Reply by seba2282** [reply]
@MajkiIT, wiesz. Jeżeli bym był mocny w tych rzeczach, to wtedy żaden problem. 🙂😉

**Reply by hawkeye116477** [reply]
A wiem jeszcze kto zapewne ma czas, no przecież Wicedyrcio @xxcriticxx, ma sporo czasu odkąd każdy sam może przydzielić etykiety :smile:

**Reply by xxcriticxx** [reply]
@hawkeye116477 nie zapominaj ze ja jestem w USA u nas bardziej z ebay korzystamy

**Reply by hawkeye116477** [reply]
To jest lista e-commerce, Ebay też się wlicza do spisu, jak chcesz to i [Amazona](https://www.amazon.de/ref=rd_www_amazon_PL/?language=pl_PL&site-redirect=PL) też możesz dodać.

Zawsze możesz też skorzystać z metody transportu opracowanej przez gargulca/Monię/[garrego](https://github.com/garry-ut99) czy jak on tam teraz się nazywa... :rofl:

**Reply by xxcriticxx** [reply]
@hawkeye116477 życzę powodzenia z amazon

**Reply by MajkiIT** [reply]
@seba2282 no początki są trudne, ale będziesz mógł się rozwijać.

**Comment by xxcriticxx** [comment]
Dodać do listy to co jest teraz i czekamy na nowe zgłoszenia

**Comment by krystian3w** [comment]
Niby nowe reklamy uruchomią:

> ### Allegro Ads: już niedługo Twoje reklamy graficzne wyświetlimy w dodatkowym miejscu
> 
> Już niedługo Twoje reklamy graficzne Allegro Ads będziemy wyświetlać dodatkowo w ofertach, pod sekcją **O sprzedającym**. Obecnie wyświetlamy je na stronie z wynikami wyszukiwania.
> 
> Dzięki tej zmianie trafisz do większej grupy kupujących i łatwiej dodasz kampanię z reklamą graficzną w panelu Allegro Ads. Nie trzeba będzie wskazywać dla niej powierzchni emisyjnej.
> 
> W związku z tą zmianą 5 listopada zaktualizujemy [regulamin Allegro Ads](#discussioncomment-107778). W tym [pliku PDF](#discussioncomment-107778) na zielono zaznaczyliśmy fragment, który dodamy.
> 
> Kliknij na [link do pliku PDF](#discussioncomment-107778) i zobacz pełny tekst po zmianie, którą wprowadzimy 5 listopada.
> 
> Pozdrawiamy
> Zespół Allegro

**Comment by krystian3w** [comment]
@maciejtarmas - popsuły się grafiki allegro smart: https://github.com/MajkiIT/polish-ads-filter/issues/17557#issue-738920017

Trzeba by sekcję zablokować lub je odblokować.

**Reply by hawkeye116477** [reply]
~A co to ma związek z tym co napisał @krystian3w....~

**Reply by seba2282** [reply]
> A co to ma związek z tym co napisał @krystian3w....

@hawkeye116477 mój błąd. Było nie na temat.
